tinylog-api-kotlin/src/main/kotlin/org/tinylog/kotlin/TaggedLogger.kt

```diff
@@ -19,25 +19,34 @@ import org.tinylog.format.AdvancedMessageFormatter
 import org.tinylog.provider.ProviderRegistry
 
 /**
- * Logger for issuing tagged log entries. Tagged loggers can be received by calling [Logger.tag].
+ * Logger for issuing tagged log entries. Tagged loggers can be received by calling [Logger.tag] or [Logger.tags].
  *
- * @param tag
+ * @param tags
  * Case-sensitive tag for logger instance
  * @see Logger.tag
+ * @see Logger.tags
  */
-class TaggedLogger internal constructor(private val tag: String?) {
+class TaggedLogger internal constructor(private val tags: Set<String?>) {
+
+	constructor(tag: String?) : this(setOf(tag))
```

Comment:
Do we really still need this constructor? If yes, could you declare the constructor as internal? I want to prohibit that users create their own instance.

Reply:
Nope, that constructor isn't needed anymore. I'll remove it 😉